using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,174 @@
+# Signal
+
+The Signal extension is bundled with the [MobileCore (Android)/ACPCore (iOS)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) extension and allows you to send postbacks to third-party endpoints and open URLs, such as web URLs or application deep links, when using rules actions in Adobe Experience Platform Launch. 
+
+To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data to a remote server.
```

Comment:
- Please change "Rules engine" to "Rules Engine" and ensure that the correct term is used throughout.

- Please change "When setting a rule" to "When creating a rule..."